app/javascript/dashboard/i18n/locale/en/helpCenter.json

```diff
@@ -324,7 +324,8 @@
         "LAST_EDITED": "Last edited"
       },
       "COLUMNS": {
-        "BY": "by"
+        "BY": "by",
+        "DELETED_AUTHOR": "Deleted Author"
```

Comment:
Instead of this can be just show `—`?

Reply:
Or maybe `System`? We are using `system` in audit logs.

ref: https://github.com/chatwoot/chatwoot/blob/develop/app/javascript/dashboard/i18n/locale/en/auditLogs.json#L22

Reply:
Updated as per the discussion.